src/EditorFeatures/VisualBasicTest/CodeActions/IntroduceVariable/IntroduceVariableTests.vb

```diff
@@ -1827,7 +1827,7 @@ Class C
         Const {|Rename:V|} As Integer = 5
         Console.WriteLine(V)
 #End ExternalSource
-    End Sub
+   End Sub
```

Comment:
This change occurred when I moved complify earlier in the IntroduceVariable logic. I didn't investigate in details.